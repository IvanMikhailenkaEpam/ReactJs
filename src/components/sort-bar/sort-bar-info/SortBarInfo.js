import React from 'react';
import * as PropTypes from "prop-types";

const SortBarInfo = props => {
  const {text} = props;
  return (
    <div>
      <div className="col-auto ml-4 font-weight-bold"><span>{text}</span></div>
    </div>
  );

};
SortBarInfo.prototype = {
  text: PropTypes.string.isRequired
};

export default SortBarInfo;
