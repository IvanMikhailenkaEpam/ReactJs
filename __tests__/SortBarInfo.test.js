import React from 'react';
import { shallow } from 'enzyme';
import SortBarInfo from "../src/components/sort-bar/sort-bar-info/SortBarInfo";

describe('SortBarInfo', () => {
  it('renders SortBarInfo correctly', () => {
    const component = shallow(<SortBarInfo text="some text" />);

    expect(component).toMatchSnapshot();
  });
});
