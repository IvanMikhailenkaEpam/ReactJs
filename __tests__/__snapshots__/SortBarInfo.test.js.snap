// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortBarInfo renders SortBarInfo correctly 1`] = `
<div>
  <div
    className="col-auto ml-4 font-weight-bold"
  >
    <span>
      some text
    </span>
  </div>
</div>
`;
